policyengine_us/variables/gov/irs/gross_income/disqualified_income.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class disqualified_income(Variable):
```

Comment:
Disqualified from what?

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/89811054/8c4813f2-2723-4576-bbb7-28628d0bf636)
The legal code mentioned the definition of investment income as the disqualified income mentioned in the internal revenue code. So I think they represent the investment income for Montana state.

Reply:
earned income disqualified income ([section 32 IRS](https://www.law.cornell.edu/uscode/text/26/32))
@Kevintowne please adjust the parameter description and label to represent that

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/3bd89ba6-4ea1-4e0f-aee2-6e13bb469a62)

Reply:
Got it. I'll fix it.